authentication/models.py

```diff
@@ -98,26 +112,71 @@ class Company(models.Model):
     startup_idea = models.TextField(blank=True)
     tags = models.CharField(max_length=255, blank=True)
 
+    @classmethod
+    def get_company(cls, *args, **kwargs):
+        return cls.objects.get(**kwargs)
```

Comment:
This exceptions should be handled from outside since our model methods basically just encapsulate objects default methods. Since we may need different approach to exception handling, the exceptions aren't handled inside this methods.